match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -0,0 +1,579 @@
+			:css('margin-bottom','-2px'):css('border-top','1px solid #DDD')
+		self:createHeader()
+	end
+)
+
+function MapVeto:createHeader()
+	self.table:tag('tr'):css('background-color','#EEE'):css('color','#000'):css('height','10px')
+		:tag('th'):css('width','33%'):done()
+		:tag('th'):css('width','34%'):css('padding','5px'):css('font-size','11px')
+		:css('text-align','center'):wikitext('Map Veto'):done()
+		:tag('th'):css('width','33%'):done()
+	return self
+end
+
+function MapVeto:vetoStart(firstVeto)
+	local textLeft = ''
+	local textCenter = ''
+	local textRight = ''
+	if firstVeto == 1 then
+		textLeft = "'''Start Map Veto'''"
+		textCenter = '[[File:Arrow sans left.svg|15x15px|link=|Left team starts]]'
+	elseif firstVeto == 2 then
+		textCenter = '[[File:Arrow sans right.svg|15x15px|link=|Right team starts]]'
+		textRight = "'''Start Map Veto'''"
+	else return self end
+	self.table:tag('tr'):css('border-bottom','5px solid #DDD'):css('background-color','#f5f5f5')
+		:css('height','10px'):css('font-size','11px')
+		:tag('th'):css('text-align','center'):css('padding','5px'):wikitext(textLeft):done()
+		:tag('th'):css('text-align','center'):css('padding','5px'):wikitext(textCenter):done()
+		:tag('th'):css('text-align','center'):wikitext(textRight):done()
+	return self
+end
+
+function MapVeto:addDecider(map)
+	if Logic.isEmpty(map) then
+		map = 'TBD'
+	else
+		map = '[['..map..'/siege|'..map..']]'
+	end
+	self.table:tag('tr'):css('border-top','1px dotted #DDD'):css('background-color','#FFF')
+			:css('height','10px'):css('font-size','11px')
+		:tag('td'):css('text-align','center'):css('font-weight','bold')
+			:tag('span'):css('color','#000'):css('background-color',Color['stay']):css('border','none')
+				:css('border-radius','0px'):css('letter-spacing','0.1em')
+				:css('font-family','\'Source Code Pro\',monospace')
+				:wikitext('DECIDER'):done()
+			:done()
+		:tag('td'):css('text-align','center'):css('padding','5px'):wikitext(map):done()
+		:tag('td'):css('text-align','center'):css('font-weight','bold')
+			:tag('span'):css('color','#000'):css('background-color',Color['stay']):css('border','none')
+				:css('border-radius','0px'):css('letter-spacing','0.1em')
+				:css('font-family','\'Source Code Pro\',monospace')
+				:wikitext('DECIDER'):done()
+			:done()
+	return self
+end
+
+function MapVeto:addRound(vetotype, map1, map2)
+	if Logic.isEmpty(map1) then
+		map1 = 'TBD'
+	else
+		map1 = '[['..map1..'/siege|'..map1..']]'
+	end
+	if Logic.isEmpty(map2) then
+		map2 = 'TBD'
+	else
+		map2 = '[['..map2..'/siege|'..map2..']]'
+	end
+	local background = ''
+	local vetoName = ''
+	if vetotype == 'ban' then
+		vetoName = 'BAN'
+		background = Color['down']
+	elseif vetotype == 'pick' then
+		vetoName = 'PICK'
+		background = Color['up']
+	elseif vetotype == 'defaultban' then
+		vetoName = 'DEFAULT BAN'
+		background = Color['dq']
+	end
+
+	self.table:tag('tr'):css('border-top','1px dotted #DDD'):css('background-color','#FFF')
```

Comment:
HARD CODED COLOR!!!!
please use css classes instead

Reply:
Moved to css